flink-end-to-end-tests/test-scripts/test_sql_client_common.sh

```diff
@@ -17,8 +17,6 @@
 # limitations under the License.
 ################################################################################
 
-set -Eeuo pipefail
-
 source "$(dirname "$0")"/common.sh
 source "$(dirname "$0")"/kafka-common.sh 0.10.2.0 3.2.0 3.2
 source "$(dirname "$0")"/elasticsearch-common.sh
```

Comment:
Have you forgotten to remove those imports?

Reply:
yes, Thanks for reminding me.